src/StatusQ/Controls/StatusPickerButton.qml

```diff
@@ -11,7 +11,13 @@ Button {
 
     property color bgColor: Theme.palette.baseColor2
     property color contentColor: Theme.palette.baseColor1
-    signal clicked()
+    property var type: StatusPickerButton.Type.NEXT
+    property int lateralMargins: 16
+
+    enum Type {
+        NEXT,
+        DOWN
+    }
```

Comment:
Just cosmetics: I think so far we've not used all-caps for type enums. If we want to stay consistent, we don't do it here either. But again, just cosmetics, no strong opinion there from my side.